internal/view/cronjob.go

```diff
@@ -64,7 +68,93 @@ func jobCtx(path, uid string) ContextFunc {
 func (c *CronJob) bindKeys(aa ui.KeyActions) {
 	aa.Add(ui.KeyActions{
 		tcell.KeyCtrlT: ui.NewKeyAction("Trigger", c.trigger, true),
+		ui.KeyShiftS:   ui.NewKeyAction("ToggleSuspend", c.toggleSuspend, true),
+	})
+}
+
+func (c *CronJob) toggleSuspend(evt *tcell.EventKey) *tcell.EventKey {
+	sel := c.GetTable().GetSelectedItem()
+	if sel == "" {
+		return evt
+	}
+
+	c.Stop()
+	defer c.Start()
+	c.showSuspendDialog(sel)
+
+	return nil
+}
+
+func (c *CronJob) showSuspendDialog(sel string) {
+	suspended := strings.TrimSpace(c.GetTable().GetCell(c.GetTable().GetSelectedRowIndex(), c.GetTable().NameColIndex()+2).Text) == "true"
```

Comment:
@ludusrusso - Tx! we should probably check the GetCell return as it can return nil if there is no cell at that location and if so return early.

Reply:
Fixed!